service/frontend/operator_handler.go

```diff
@@ -368,6 +368,42 @@ func (h *OperatorHandlerImpl) DeleteWorkflowExecution(ctx context.Context, reque
 	return &operatorservice.DeleteWorkflowExecutionResponse{}, nil
 }
 
+// AddOrUpdateRemoteCluster adds or updates the connection config to a remote cluster.
+func (h *OperatorHandlerImpl) AddOrUpdateRemoteCluster(
+	ctx context.Context,
+	request *operatorservice.AddOrUpdateRemoteClusterRequest,
+) (_ *operatorservice.AddOrUpdateRemoteClusterResponse, retError error) {
+	return nil, serviceerror.NewUnimplemented("TODO: Need to get from another PR")
```

Comment:
are these ready yet or should we just leave them like this?

Reply:
They don't exist yet in master (but do in the API at the version I need). I think they need to be left like this for now